webroot/src/store/license/license.spec.ts

```diff
@@ -414,4 +414,67 @@ describe('License Store Getters', async () => {
 
         expect(licensee).to.matchPattern(record);
     });
+    it('should successfully get privilege by LicenseeId And PrivilegeId', async () => {
+        const licensee1 = {
+            id: '1',
+            privileges: [
+                new License({ id: '1' }),
+                new License({ id: '2' }),
+            ]
+        };
+
+        const licensee2 = {
+            id: '2',
+            privileges: [
+                new License({ id: '12' }),
+                new License({ id: '22' }),
+            ]
+        };
+        const state = { model: [ licensee1, licensee2 ] };
+        const privilege = getters.getPrivilegeByLicenseeIdAndId(state)({ licenseeId: '2', privilegeId: '12' });
+
+        expect(privilege.id).to.equal('12');
+    });
+    it('should successfully not get privilege by LicenseeId And PrivilegeId (no licensee)', async () => {
+        const licensee1 = {
+            id: '1',
+            privileges: [
+                new License({ id: '1' }),
+                new License({ id: '2' }),
+            ]
+        };
+
+        const licensee2 = {
+            id: '2',
+            privileges: [
+                new License({ id: '12' }),
+                new License({ id: '22' }),
+            ]
+        };
+        const state = { model: [ licensee1, licensee2 ] };
+        const privilege = getters.getPrivilegeByLicenseeIdAndId(state)({ licenseeId: '3', privilegeId: '12' });
+
+        expect(privilege).to.equal(null);
+    });
+    it('should successfully not get privilege by LicenseeId And PrivilegeId (no privilege)', async () => {
+        const licensee1 = {
+            id: '1',
+            privileges: [
+                new License({ id: '1' }),
+                new License({ id: '2' }),
+            ]
+        };
+
+        const licensee2 = {
+            id: '2',
+            privileges: [
+                new License({ id: '12' }),
+                new License({ id: '22' }),
+            ]
+        };
+        const state = { model: [ licensee1, licensee2 ] };
+        const privilege = getters.getPrivilegeByLicenseeIdAndId(state)({ licenseeId: '2', privilegeId: '123' });
```

Comment:
Perhaps change this privilege ID to be one available in licensee with ID 1 to make this a bit more realistic?
e.g.

```suggestion
        const privilege = getters.getPrivilegeByLicenseeIdAndId(state)({ licenseeId: '2', privilegeId: '1' });
```